atdgen/src/ov_mapping.ml

```diff
@@ -307,13 +309,14 @@ let def_of_atd is_shallow (loc, (name, param, an), x) =
   let o =
     match as_abstract x with
     | Some (_, an2) ->
-        Ocaml.get_ocaml_module_and_t Validate name an
-        |> Option.map (fun (types_module, main_module, ext_name) ->
-          let args = List.map (fun s -> Tvar (loc, s)) param in
-          External (loc, name, args,
-                    Ocaml.Repr.External (types_module, main_module, ext_name),
-                    (Validate.get_validator an2, false))
-        )
+        (match Ocaml.get_ocaml_module_and_t Validate name an with
```

Comment:
The best I can tell, there seem to be two notions of "abstract" in the code base. This change allows for `type raw_json = abstract` to work as expected while (I think) still respecting the existing `as_abstract` notion.